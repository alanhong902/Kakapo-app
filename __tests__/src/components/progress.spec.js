import React from 'react';
import renderer from 'react-test-renderer';
import Progress from 'components/ui/progress/progress';

test('<Progress/> render', () => {
  const tree = renderer.create(<Progress />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('<Progress/> rounds 0.415 to 42% ', () => {
  const tree = renderer.create(<Progress {...{ progress: 0.415 }} />).toJSON();
  expect(tree).toMatchSnapshot();
});
