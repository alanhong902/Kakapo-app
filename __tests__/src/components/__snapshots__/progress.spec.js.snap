// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Progress/> render 1`] = `
<div
  className="progress"
>
  <div
    className="progress-barberpole"
    style={
      Object {
        "width": "0%",
      }
    }
  />
  <div
    className="progress-text"
  >
    0
    %
  </div>
</div>
`;

exports[`<Progress/> rounds 0.415 to 42%  1`] = `
<div
  className="progress"
>
  <div
    className="progress-barberpole"
    style={
      Object {
        "width": "42%",
      }
    }
  />
  <div
    className="progress-text"
  >
    42
    %
  </div>
</div>
`;
